### PR TITLE
br: update doc for encrypt-br

### PR DESCRIPTION
**Pull request by joccau**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -232,7 +232,8 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 > **警告：**
 >
-> 如密钥丢失，备份的数据将无法恢复到集群中。
+> - 密钥丢失，备份的数据将无法恢复到集群中。
+> - 加密功能需在 br 工具 和 TiDB 集群都不低于 v5.3.0 的版本上使用，且加密备份得到的数据无法在低于 v5.3.0 版本的集群上恢复
```
Comment (joccau) [inline br/use-br-command-line-tool.md:236]: Need "。" at the end of sentence.

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -232,7 +232,8 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 > **警告：**
 >
-> 如密钥丢失，备份的数据将无法恢复到集群中。
+> - 密钥丢失，备份的数据将无法恢复到集群中。
+> - 加密功能需在 br 工具 和 TiDB 集群都不低于 v5.3.0 的版本上使用，且加密备份得到的数据无法在低于 v5.3.0 版本的集群上恢复。
```
Comment (Liuxiaozhen12) [inline br/use-br-command-line-tool.md:236]:
```suggestion
> - 加密功能需在 br 工具和 TiDB 集群都不低于 v5.3.0 的版本上使用，且加密备份得到的数据无法在低于 v5.3.0 版本的集群上恢复。
```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

**Comment by Liuxiaozhen12** [comment]
@joccau Please involve a tech review to give one approval, thx!

**Review by kennytm — APPROVED** [review]

**Pull request merged**